### DISCUSSION TITLE
Keycloak 22.0.1- Admin Page UI and Lables are different when logged in with Non Admin User

### DISCUSSION THREAD

**Discussion by a9ki21023** [keycloak/keycloak#23051] [Q&A]
Hello Everyone,

I am using Keycloak Version 22.0.1 FROM quay.io/keycloak/keycloak:22.0.1. I noticed problem with UI when i login with a Non Admin User. Example screenshots are below.

**KeyCloak Non Admin Login :**
![relam-user_screenshot](https://github.com/keycloak/keycloak/assets/139567993/6d4a84f1-b11a-42ef-b43c-eb79846a07db)

**KeyCloak Admin Login:**
![Keycloak Admin](https://github.com/keycloak/keycloak/assets/139567993/2b2fad83-0b0a-4453-ba0e-cd60282b50a2)

**Problems:**
The Lables are named as clientScope instead of Client Scope
The clientScope contains protocol which is protocolTypes.openIdConnect instead of OpenIDConnect
The clientScope Explaination is clientScopeExplain Learn More

**NOTE:** The above problems are noticed only with a Realm User and not the Admin User. If i assign the admin Role Mapping to this user the UI Looks Good. Can someone confirm if this problem is introduced in Keycloak 22.0.1 ? And is there is a workaround ? 

**Steps to Repro:**
1. Pull docker container  FROM quay.io/keycloak/keycloak:22.0.1
2. Login to Keycloak Admin Page with admin user
3. Create a new Realm Ex: Test
4. Create a new User is Master Ex: TestUser, Generate Password.
5. Assign the Realm Policies to the TestUser
6. Login with TestUser and Select the Realm Test. Notice the UI is different and Labels are different.

**Comment by ssilvert** [comment]
I'm not seeing this problem.  What is happening in your case is that the message bundles are missing or not applied.  But I don't know why it is happening.

I'm not using Docker, but I don't see why that would matter.

**Comment by ssilvert** [comment]
This was logged as an issue at https://github.com/keycloak/keycloak/issues/23052.  So I'm closing the discussion.
